synapse/lib/modelrev.py

```diff
@@ -1205,3 +1206,753 @@ async def _propArrayToForm(self, layers, propfull, formname):
         }
         '''
         await self.runStorm(storm, opts=opts)
+
+class ModelMigration_0_2_31:
+    @classmethod
+    async def anit(cls, core, layers):
+        self = cls()
+
+        self.core = core
+        self.layers = layers
+
+        self.meta = {'time': s_common.now(), 'user': self.core.auth.rootuser.iden}
+
+        self.editcount = 0
+        self.nodeedits = {}
+
+        self.nodes = await s_spooled.Dict.anit(dirn=self.core.dirn)
+        self.todos = await s_spooled.Set.anit(dirn=self.core.dirn)
+
+        self.core.onfini(self.nodes)
```

Comment:
We don't need these to hang out for the lifetime of the cortex right?  Shouldn't they be fini'd when the migration is complete?

Reply:
They both get fini'd at the end of the migration (around L1643). I added this just in case the migration doesn't complete for some reason.